src/components/App.js

```diff
@@ -1,16 +1,89 @@
 import React from 'react';
+import Homepage from './Homepage'
+import Basket from './Basket'
+import MenuResults from './MenuResults'
 
 import '../styles/App.scss';
 
 class App extends React.Component {
   constructor(){
     super();
+    this.state = {
+      menu: {},
+      order: {},  //to add empty history of order
+      
+  
+    }
+    this.getOrder = this.getOrder.bind(this);
+    // this.fetchMenu = this.fetchMenu.bind(this)
+    this.sendOrder = this.sendOrder.bind(this)
+    
   }
 
+  componentDidMount(){
+    this.fetchMenu()
+  }
+
+  fetchMenu(){
+    fetch('/menu')
+    .then(response => response.json())
+    .then(body => {
+      console.log(body)
+      this.setState({
+        menu: body,
+      })
+  })
+}
+
+sendOrder(){
+  this.setState({
```

Comment:
Does `sendOrder` not send data to server?

Reply:
Just saw that the order gets sent from Basket. It might better to do fetch and clear data in one go so that you only clear order data once you know that the API request has been successful